doc-content/enterprise-only/processes/viewing-process-instances-variables-advanced-filters-in-bc-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='viewing-process-instances-variables-advanced-filters-in-bc-proc']
+= Viewing process instance variables using advanced filters
+
+You can use the *Advanced Filters* option in {CENTRAL} to view process instance variables. When you create a filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the advanced filters icon to expand the *Advanced Filters* panel.
+. In the *Advanced Filters* panel, enter the name and description of the filter, and click *Add New*.
+. From the *Select column* list, select the `processId` attribute. The value will change to *processId != value1*.
+. From the *Select column* list, select *equals to* for the logical query.
+. In the text field, enter the name of the process id.
+. Click *Save* and the filter is applied on the current process instance list.
+. Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed.
```

Comment:
s/Click *Show/hide columns* on the upper right of the process instances list and the process instance variables of the specified process id will be displayed./Click *Show/hide columns* on the upper right of the process instances list. The process instance variables of the specified process id is displayed.

Reply:
Done.